src/components/side_panel/pivot/pivot_layout_configurator/add_dimension_button/add_dimension_button.xml

```diff
@@ -1,8 +1,7 @@
 <templates>
   <t t-name="o-spreadsheet-AddDimensionButton">
-    <span class="btn btn-sm btn-link add-dimension" t-on-click="togglePopover" t-ref="button">
-      Add
-    </span>
+    <!-- ADM TODO not a fan of this one -->
```

Comment:
:eyes:

Reply:
Francois was fine with the button so 🤷 I definitely need to remove the comment tho 😅